handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -162,6 +164,14 @@ public SslContextBuilder sslProvider(SslProvider provider) {
         return this;
     }
 
+    /**
+     * Sets the Keystore type that should be used. {@code null} uses the default one.
```

Comment:
s/Keystore/KeyStore/. Would probably be useful to make it as `{@link KeyStore}` as well.